File: Medium/42MissingNumber.py
def missingNumbers(nums):
    # Write your code here.
    includeNumbers = set(nums)
    solution = []
    for num in range(1, len(nums)+3):
        if not num in includeNumbers:
            solution.append(num)

    return solution


print(missingNumbers([1, 2, 4]))

File: Medium/42MissingNumber.js
function missingNumbers(nums) {
  let includeNumbers = new Set(nums);
  let solution = [];
  for (let i = 1; i < nums.length + 3; i++) {
    if (!includeNumbers.has(i)) {
      solution.push(i);
    }
  }
  return solution;
}

console.log(missingNumbers([1, 2, 4]));
